src/main/java/de/tum/in/www1/artemis/service/iris/dto/IrisCombinedLectureIngestionSubSettingsDTO.java

```diff
@@ -0,0 +1,4 @@
+package de.tum.in.www1.artemis.service.iris.dto;
+
+public record IrisCombinedLectureIngestionSubSettingsDTO(boolean enabled) {
```

Comment:
Why do we need a DTO that just wraps around one boolean?

Reply:
More fields will be added in the future...